system-variables.md

```diff
@@ -439,6 +439,13 @@ This variable is an alias for `last_insert_id`.
 - Unit: Rows
 - The maximum number of rows returned by the `SELECT` statements.
 
+### require_primary_key <span class="version-mark">New in TBD</span>
```

Comment:
I am not sure, at what version this will become available, it has been merged already. So I think it will be included in the next cut for next version.

But someone need to confirm it for me. 🙏